almond/build.json

```diff
@@ -1,12 +1,11 @@
 {
   "build_from": {
-    "aarch64": "homeassistant/aarch64-base:3.13",
-    "amd64": "homeassistant/amd64-base:3.13",
-    "i386": "homeassistant/i386-base:3.13",
-    "armv7": "homeassistant/armv7-base:3.13",
-    "armhf": "homeassistant/armhf-base:3.13"
+    "aarch64": "homeassistant/aarch64-base-debian:buster",
+    "amd64": "homeassistant/amd64-base-debian:buster",
+    "armhf": "homeassistant/armhf-base-debian:buster",
+    "armv7": "homeassistant/armv7-base-debian:buster"
```

Comment:
Why move away from Alpine?

Reply:
Some of the prebuilt software we use depends on glibc.

Reply:
So why not rebuild it?

Reply:
I think the issue was snowboy (which uses glibc symbols on arm64) - that's a binary blob, there is nothing we can do about it.

Reply:
Why are we adding a closed source in here? Are we allowed to distribute it even?
How does this work for the future? Snowboy has been shutdown?

Reply:
Yes snowboy is distributable. Snowboy is currently discontinued but it works. There are no open-source wake-word capabilities (Picovoice, used by Ada, is not open-source either, and it has no good general wake word).
In the future, we'll likely change the wakeword and the implementation, but it probably won't be open-source either.